### ISSUE TITLE
About slow charging problem 

### ISSUE THREAD

**Issue by Mustafa7822** [PixelExperience/android-issues#4484]
![Screenshot_20230217-015656_Pixel Launcher](https://user-images.githubusercontent.com/119671334/219482615-d6032975-2c23-4cf9-b16e-f0db4b9a622e.png)
![Screenshot_20230217-015603_Pixel Launcher](https://user-images.githubusercontent.com/119671334/219482744-1d8b0c1e-bc1c-4a20-98ef-68cb7ce2c513.png)

## Build date

PixelExperience_Plus_miatoll-13.0-20230108-0555-OFFICIAL

## Expected Behavior

Device should be charged rapidly as per their functionality 

## Current Behavior

As I know that sometimes it happens not in every 3 to 4 days 
 but nowadays it is happening in every 2 or 3 days when we connect charger to device then he says charging slowly or it shows only charging this is happening only after update of January build. I attached some screenshots for explaining much easier 

## Possible Solution

I don't know 

## Steps to Reproduce

1.first of all you have to wait on lock screen until not recognising face 
2.then after limited period of face recognisation you have to connect charger plug
3.after that you can see what's happening 
4.

/device miatoll 
/version thirteen_plus

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by dereference23** [comment]
Duplicate of #4353

**Comment by pixelexperiencebot** [comment]
Issue closed by maintainer request.
